README.md

```diff
@@ -25,3 +27,31 @@ JdbcStorageFactory.newBuilder()
                   .setDataSource(dataSource)
                   .build();
 ```
+
+### Type mapping
+
+Data types differ for various SQL databases. There is `TypeMapping` to deal with it.
+The mapping defines correspondence of `Type` to a name for a particular database. 
+`Type` is an abstraction for a data type in a database. 
+
+The type mapping is selected automatically basing on the JDBC connection string.
+If there is no standard mapping for the database, MySQL mapping will be used as the default.
+
+If the automatically selected mapping doesn't match your requirements,
+a custom mapping can be specified during creation of `JdbcStorageFactory`.
+
+A custom mapping should define names for all `Type`s, except `Type.ID`:
+
+```
+TypeMapping mapping = TypeMapping.newBuilder()
+                                 .add(Type.BYTE_ARRAY, "*")
+                                 .add(Type.INT, "*")
+                                 .add(Type.LONG, "*")
+                                 .add(Type.STRING_255, "*")
+                                 .add(Type.STRING, "*")
+                                 .add(Type.BOOLEAN, "*")
+                                 .build();
+JdbcStorageFactory.newBuilder()
+                  .setTypeMapping(mapping)
+``` 
+
```

Comment:
Please get rid of redundant empty lines here and in similar cases.

Reply:
Done.